### ISSUE TITLE
Share domains between user and team accounts

### ISSUE THREAD

**Issue by JounQin** [vercel/vercel#3671]
`The domain prettier.1stg.me is already owned by another ZEIT account.`

User Account: https://zeit.co/jounqin
Team Account: https://zeit.co/rxts

**Comment by styfle** [comment]
Hi @JounQin 

You can move a domain from a user to a team by visiting the domain dashboard https://zeit.co/dashboard/domains

Then select the three dots on the far right and select "Move" in the popover menu.

![image](https://user-images.githubusercontent.com/229881/73312159-13502080-41f6-11ea-9855-96defc542de7.png)

**Comment by JounQin** [comment]
@styfle Can the user account continue bind the domain after moving it to a team?

**Comment by styfle** [comment]
@JounQin No, the domain must be associate to a user or a team but not both.

Can you explain your use case so we can better understand if we need a new feature?

**Comment by JounQin** [comment]
@styfle As the issue content, I do have a personal account and a team account, I want to share the same domain, because some projects are personal while some are under team account.

For instance, https://blog.1stg.me/ is my personal blog, and https://prettier-plugins.now.sh is a @rx-ts [project](https://github.com/rx-ts/prettier), I want to add an alias https://prettier.1stg.me, but it seems impossible now.

**Comment by styfle** [comment]
Ok thanks for sharing!

We have future plans for more granular permissions so that a team could have sub-teams with access to specific projects (a team would act more like an org).

We don't have an ETA for this feature but I'll forward your feedback, thanks!

**Comment by mightymatth** [comment]
> @JounQin No, the domain must be associate to a user or a team but not both.
> 
> Can you explain your use case so we can better understand if we need a new feature?

2 years later, we've encountered this terrific Vercel limitation. 

Of course we need this feature. It's like if, in a 4-flat building, you give permission to a single flat to use electricity and forbid every other flat from using it.

We've one client (a big organization) who gave us to deploy the app on their subdomain, let's say `ourapp.client.com`. We've registered **only this subdomain** on Vercel.
A few weeks after that, there is another team that wants to deploy their app on the organization's subdomain (also using Vercel), let's say `theirapp.client.com`. What do they get? The error saying that another Vercel account uses this subdomain. 
When we check the Domain section in Vercel, we can see that you've added a wildcard in the DNS CNAME, even though we added just 1 subdomain. This way reserved the entire space and nobody else can use `*.client.com` despite us. This is absurd!

Excuse me if it sounds offensive, but I just want to show the importance of the issue.
